### ISSUE TITLE
Increase contrast for .caption class to meet WCAG AA 1.4.3

### ISSUE THREAD

**Issue by aaroncole** [SU-SWS/open_framework#310]
The current color for .caption fails WCAG AA 1.4.3.

Guideline:
https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

WebAIM Color Contrast Tool:
http://webaim.org/resources/contrastchecker/
